libraries/AP_NavEKF2/AP_NavEKF2_AirDataFusion.cpp

```diff
@@ -39,6 +41,9 @@ void NavEKF2_core::FuseAirspeed()
     float SK_TAS;
     Vector24 H_TAS;
     float VtasPred;
+    Vector28 Kfusion; // Kalman gain vector
```

Comment:
this change exposes what looks like an existing bug in airdata fusion. At line 138 we use Kfusion[24] but that is never initialised. 

../../libraries/AP_NavEKF2/AP_NavEKF2_AirDataFusion.cpp: In member function ‘void NavEKF2_core::FuseAirspeed()’:
../../libraries/AP_NavEKF2/AP_NavEKF2_AirDataFusion.cpp:143:60: error: ‘*((void*)& Kfusion +96)’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
                 statesArray[j] = statesArray[j] - Kfusion[j] * innovVtas;
                                                   ~~~~~~~~~^

Reply:
this turns out to be a false alarm in the compiler, as the array index limit never goes above 23
we'll need to add some code to convince the compiler this is OK